### ISSUE TITLE
Polymorphic relationships for namespaced models break rails_admin.js

### ISSUE THREAD

**Issue by tosbourn** [sferik/rails_admin#3263]
When using polymorphic associations like;

```ruby
class Region < ApplicationRecord
  belongs_to :contentable, polymorphic: true
end

class Event::Exhibition < ApplicationRecord
  has_many :regions, as: :contentable
end

class Post < ApplicationRecord
  has_many :regions, as: :contentable
end
```

Everything works bar the ajax fetch of instances of namespaced models.

When I try and select an `Event::Exhibition`, for example, I see this in my browsers console.
```
Error: Syntax error, unrecognized expression: #event::exhibition-js-options rails_admin.js:1502:8
    error http://localhost:3000/assets/rails_admin/rails_admin.js:1502
```

When I select `Post` all of my posts are returned as expected.

Is this a bug or should I be able to work around this with different settings? The only config I have for this is to tell Rails Admin to use the field.

```ruby
edit do
  field :contentable
end
```

**Comment by tosbourn** [comment]
After some digging, I've spotted that in my setup my relationships generate the following select values;

``` html
<option value=""></option>
<option value="Event::ArtFair">Art fair</option>
<option value="Event::Exhibition">Exhibition</option>
<option selected="selected" value="Post">News article</option>
```

Sizzle doesn't like the `::` and falls over.

In my code inspector I can escape `::` to `\:\:`, when I do this, things work as expected.

I found [someone else talking about Sizzle](https://stackoverflow.com/questions/25484580/selecting-an-element-with-id-having-colons-in-sizzle-jquery) choking on `:`.

**Comment by DanAndreasson** [comment]
@tosbourn did you figure out a good solution for this?

**Comment by tosbourn** [comment]
@DanAndreasson, unfortunately not. We had a hacky JS-based fix to escape those values and then (unrelated to this bug) migrated away from name-spaced models like this.

**Comment by mshibuya** [comment]
Fixed by #3377.
